packages/cli/src/cmds/compare/ChangeReporter.ts

```diff
@@ -117,36 +122,24 @@ export default class ChangeReporter {
     public workingDir: string,
     public srcDir: string
   ) {
-    this.appmapData = new AppMapData(workingDir);
-    this.appmapIndex = new AppMapIndex(this.appmapData);
+    this.paths = new Paths(workingDir);
+    this.digests = new Digests(this.paths);
   }
 
   async initialize() {
-    await this.appmapIndex.build();
+    await this.digests.build();
 
     this.baseManifest = JSON.parse(
-      await readFile(this.appmapData.manifestPath(RevisionName.Base), 'utf-8')
+      await readFile(this.paths.manifestPath(RevisionName.Base), 'utf-8')
     );
     this.headManifest = JSON.parse(
-      await readFile(this.appmapData.manifestPath(RevisionName.Head), 'utf-8')
+      await readFile(this.paths.manifestPath(RevisionName.Head), 'utf-8')
     );
 
-    await this.loadMetadata();
-
-    const baseAppMaps = new Set(await this.appmapData.appmaps(RevisionName.Base));
-    const headAppMaps = new Set(await this.appmapData.appmaps(RevisionName.Head));
-    this.newAppMaps = [...headAppMaps].filter((appmap) => !baseAppMaps.has(appmap));
-    this.changedAppMaps = [...headAppMaps]
-      .filter((appmap) => baseAppMaps.has(appmap))
-      .map((appmap) => ({ appmap }));
```

Comment:
Note that every AppMap that appeared in both `base` and `head` was considered "changed". Ooops. Now an AppMap is only "changed" if it's a test, it appears in both `base` and `head`, and the `digest` has changed.